align_anything/configs/template.py

```diff
@@ -1273,104 +1163,6 @@ def format_sample(self, raw_sample: dict[str, Any], path: str = None) -> dict[st
     def check_equal(self, raw_sample: dict[str, Any]) -> bool:
         return False
 
-@register_template('Qwen2-VL')
```

Comment:
If this Template is deleted, should the content in the [Model-Zoo](https://github.com/PKU-Alignment/align-anything/blob/main/Model-Zoo.md) be modified, or should the [Model-Zoo](https://github.com/PKU-Alignment/align-anything/blob/main/Model-Zoo.md) be deleted?